react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import LoginForm from '../Login/components/LoginForm';
+
+class Login extends Component {
+  submit = values => {
+    //should verified email and password
+    this.props.history.push("/game");
+  };
+
+  getInitialValues() {
```

Comment:
Non life-cycle methods should be set as arrow function. This is a convention we follow.
Also, this initial values doesn't need to be set by a method. A constant should be enough.
```js
// ../Login/constants.js
export const INITIAL_VALUES = { email: '', password: '' };
// ../Login/index.js
return (
      <LoginForm
        onSubmit={this.submit}
        initialValues={INITIAL_VALUES}
      />
  );
```
**Another way, and i think is preferable, define default props in `LoginForm` component.**